lib/file.js

```diff
@@ -137,28 +138,30 @@ internals.prepare = async function (response) {
 
 internals.marshal = async function (response) {
 
-    if (response.source.settings.lookupCompressed &&
-        !response.source.settings.start &&
-        response.source.settings.end === undefined &&
-        response.request.server.settings.compression !== false) {
+    const { request, source } = response;
+    const { settings } = source;
 
-        const lookupMap = response.source.settings.lookupMap || internals.defaultMap;
-        const encoding = response.request.info.acceptEncoding;
+    if (settings.lookupCompressed &&
+        !settings.start &&
+        settings.end === undefined &&
+        request.server.settings.compression !== false) {
+
+        const lookupMap = settings.lookupMap || internals.defaultMap;
+        const encoding = request.info.acceptEncoding;
         const extension = lookupMap.hasOwnProperty(encoding) ? lookupMap[encoding] : null;
         if (extension) {
-            const precompressed = new Fs.File(`${response.source.path}${extension}`);
-            let stat;
+            const precompressed = new Fs.File(`${source.path}${extension}`);
             try {
-                stat = await precompressed.openStat('r');
+                var stat = await precompressed.openStat('r');
```

Comment:
Why not just keep the `let`?

Reply:
For consistency with the hapi codebase.

Reply:
Alright. I thought the `var` usage was just for the `catch` block.